### PR TITLE
cm-3

### PR DESCRIPTION
**Pull request by J2thatsme**
Could not write a test without data

### CODE REVIEW

File: types/comtac/CM-3/lifecycle.schema.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+  "$id": "https://akenza.io/comtac/CM-3.lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Comtac CM-3",
+  "type": "object",
+  "properties": {
+    "tempPt100": {
```
Comment (mariusgiger) [inline types/comtac/CM-3/lifecycle.schema.json:7]: this should not be in here -> move to default?

File: types/comtac/CM-3/lifecycle.schema.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      "title": "Temperature PT100",
+      "description": "TEMP PT100 placed",
+      "type": "boolean"
+    },
+    "tempI2C": {
```
Comment (mariusgiger) [inline types/comtac/CM-3/lifecycle.schema.json:12]: this should not be in here -> move to default?

File: types/comtac/CM-3/lifecycle.schema.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      "title": "ACC",
+      "description": "ACC placed",
+      "type": "boolean"
+    },
+    "extMEM": {
+      "title": "EXT MEM",
+      "description": "EXT MEM placed",
+      "type": "boolean"
+    },
+    "lastTempValid": {
```
Comment (mariusgiger) [inline types/comtac/CM-3/lifecycle.schema.json:27]: this should not be in here -> move to default?

File: types/comtac/CM-3/meta.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "LPN CM-3",
+  "version": "1.0.0",
+  "manufacturer": "Comtac",
+  "url": "https://www.comtac.ch/de/produkte/lora/condition-monitoring/lpn-cm-3-sensor.html",
+  "description": "The LPN CM-3 is able to read temperature.",
+  "author": "Akenza AG",
+  "firmwareVersion": "V0.0.2",
```
Comment (mariusgiger) [inline types/comtac/CM-3/meta.json:8]: correct firmware version?

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please add a test

File: types/comtac/CM-3/lifecycle.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/comtac/CM-3/lifecycle.schema.json:7]: These are only flags

File: types/comtac/CM-3/lifecycle.schema.json (hunk 2 above)
Comment (J2thatsme) [reply types/comtac/CM-3/lifecycle.schema.json:12]: These are only flags

File: types/comtac/CM-3/lifecycle.schema.json (hunk 3 above)
Comment (J2thatsme) [reply types/comtac/CM-3/lifecycle.schema.json:27]: These are only flags

File: types/comtac/CM-3/meta.json (hunk 4 above)
Comment (J2thatsme) [reply types/comtac/CM-3/meta.json:8]: yes

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**